packages/dataviews/src/components/dataviews-view-config/style.scss

```diff
@@ -0,0 +1,43 @@
+.dataviews-view-config {
+	width: 363px;
```

Comment:
Let's make this a bit narrower for now, and align to the 4px baseline. `320px` works well.